ditail.py

```diff
@@ -129,6 +145,8 @@ def tail_files_in(directory):
 def create_argparser():
     parser = argparse.ArgumentParser(description="Prints new lines added to all files in a directory and its subdirectories. Any files added after startup will also be monitored.")
     parser.add_argument("-d", "--debug", action="store_true", help="Enable logging (at DEBUG level)")
+    parser.add_argument("-f", "--filter", action="store", required=False, help="Filter filename (regexp)")
+    parser.add_argument("--disable-truncate-detect", action="store_true", help="Disable file truncate detection")
```

Comment:
The help text needs to avoid just repeating the argument name. It should describe what truncate detection is, but I've forgotten :smile: . It probably should have short form too, assuming it's not used very rarely.